opm/simulators/wells/GasLiftGroupInfo.hpp

```diff
@@ -171,7 +171,7 @@ class GasLiftGroupInfo
     GroupIdxMap group_idx_;
     int next_group_idx_ = 0;
     // Optimize only wells under THP control
-    bool optimize_only_thp_wells_ = true;
+    bool optimize_only_thp_wells_ = false;
 
```

Comment:
This significantly slows down the well assembly time

Reply:
I have made a small fix that reduce most of the extra time. Most of the time is spent in `computeWellRatesWithBhp` Currently the code passes evaluation objects of size 7 to this function. Since the derivatives are not needed for this call this leads to a significant uneccesary overhead in the computational time. Improving this shouldn't be too hard. I will put it on my TODO list. But probably not before summer.